pkg/cmd/target/target.go

```diff
@@ -19,4 +19,5 @@ func init() {
 	TargetCmd.AddCommand(TargetSetCmd)
 	TargetCmd.AddCommand(targetRemoveCmd)
 	TargetCmd.AddCommand(targetSetDefaultCmd)
+	TargetSetCmd.Flags().StringVarP(&pipeFile, "file", "f", "", "Path to JSON file for target configuration, use '-' to read from stdin")
```

Comment:
This needs to go in the `init` function of the set command

Reply:
one more things i should add this in the `set.go` to make things more organised right 
```go
func init() {
	TargetSetCmd.Flags().StringVarP(&pipeFile, "file", "f", "", "Path to JSON file for target configuration, use '-' to read from stdin")
}
```